addons/sys_core/fnc_isPluginConnected.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: Killerswin2
+ * Check if the plugin is connected.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * Plugin is connected <BOOL>
+ *
+ * Example:
+ * [] call acre_sys_core_fnc_isPluginConnected
+ *
+ * Public: No
+ */
+
+// check that server started, no pipe errors, and no generally errors in io
+GVAR(connectedPlugin) = (EGVAR(sys_io,pipeCode) == "1" && !EGVAR(sys_io,hasErrored) && EGVAR(sys_io,serverStarted));
```

Comment:
```suggestion
// check that server started, no pipe errors, and generally no errors in io
GVAR(connectedPlugin) = (EGVAR(sys_io,pipeCode) == "1" && {!EGVAR(sys_io,hasErrored) && {EGVAR(sys_io,serverStarted)}});
```

Reply:
can I ask why you recommend I added more scopes?

Reply:
Lazy evaluation; no point in making checks that couldn't be true when the pipe ain't working.